src/logiDice.js

```diff
@@ -73,13 +73,16 @@ module.exports = {
 			return Promise.all(recPromises).then((subQueries) => {
 				let diceResult = subQueries.reduce((tally, current, index) => {
 					return mergeResults(tally, current);
-				}, false);
+				}, {
+					rolls: [],
```

Comment:
i couldn't figure out a good test for the initial guard that didn't break something else...... so now there's a new, more specific guard!